bl/bluart.c

```diff
@@ -49,9 +54,9 @@ void uart_init(ctx_t *ctx) {
 #endif
 
 #ifdef LL_USART_FIFOTHRESHOLD_1_8
```

Comment:
I assume this commenting these out is fine?

Reply:
yeah, will delete; FIFO is off in CR1